app-modules/authorization/src/Filament/Resources/RoleResource/Pages/ListRoles.php

```diff
@@ -100,14 +100,14 @@ public function table(Table $table): Table
             ])
             ->actions([
                 Action::make('duplicateRole')
-                    ->label('Duplicate')
```

Comment:
Are we sure we want to remove this `label` here? That changes the label of the table action button from `Duplicate` to `Duplicate role`. Which I don't believe was asked for in this ticket.